src/ui/src/writerApi.ts

```diff
@@ -244,6 +246,134 @@ export class WriterApi {
 
 		return key ? `${url}/${key}` : url;
 	}
+
+	async fetchMcpConnectedApps(
+		orgId: number,
+		appId?: string,
+	): Promise<WriterApiMcpApp[]> {
+		const url = new URL(
+			`/api/mcp-gateway/v2/organization/${orgId}/app-configurations`,
+			"https://app.qordobadev.com",
```

Comment:
Maybe we can use dev with a feature flag?

Reply:
You just need to use `this.#baseUrl`, it points to the right environment. 

```suggestion
			this.#baseUrl,
```

For local dev, you need to set this environment variable.
```sh
export VITE_WRITER_BASE_URL=https://app.qordobadev.com
```